javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -1213,6 +1214,95 @@ public static CharSeq charSeq(CharSequence sequence) {
         return TreeSet.ofAll(comparator, elements);
     }
 
+    /**
+     * Alias for {@link LinkedHashMap#of(Object, Object)}
+     *
+     * @param <K>   The key type.
+     * @param <V>   The value type.
+     * @param key   A singleton map key.
+     * @param value A singleton map value.
+     * @return A new LinkedHashMap instance containing the given entry
+     */
+    public static <K, V> LinkedHashMap<K, V> linkedHashMap(K key, V value) {
+        return LinkedHashMap.of(key, value);
+    }
+
+    /**
+     * Alias for {@link LinkedHashMap#ofAll(Map)}
+     *
+     * @param <K> The key type.
+     * @param <V> The value type.
+     * @param map A map entry.
+     * @return A new LinkedHashMap instance containing the given map
+     */
+    public static <K, V> LinkedHashMap<K, V> linkedHashMap(Map<? extends K, ? extends V> map) {
+        return LinkedHashMap.ofAll(map);
+    }
+
+    /**
+     * Alias for {@link HashMap#of(Object, Object)}
+     *
+     * @param <K>   The key type.
+     * @param <V>   The value type.
+     * @param key   A singleton map key.
+     * @param value A singleton map value.
+     * @return A new HashMap instance containing the given entry
+     */
+    public static <K, V> HashMap<K, V> hashMap(K key, V value) {
```

Comment:
We (will) have conversion methods `toHashMap()` instead of `toMap()`, etc.

But the aliases in fact should be the simple names, e.g.

* `Map` instead of `hashMap`
* `SortedMap` instead of `TreeMap`
* ...